Formula/folly.rb

```diff
@@ -41,21 +39,19 @@ def install
     ENV.cxx11
 
     cd "folly" do
-      # Workaround for "no matching function for call to 'clock_gettime'"
-      # See upstream PR from 2 Oct 2016 facebook/folly#488
-      if DevelopmentTools.clang_build_version >= 800
+      if MacOS.version == "10.11" && MacOS::Xcode.installed? && MacOS::Xcode.version >= "8.0"
+        # Workaround for "no matching function for call to 'clock_gettime'"
+        # See upstream PR from 2 Oct 2016 facebook/folly#488
         inreplace ["Benchmark.cpp", "Benchmark.h"] do |s|
-          s.gsub! "detail::DEFAULT_CLOCK_ID",
-                  "(clockid_t)detail::DEFAULT_CLOCK_ID"
           s.gsub! "clock_gettime(CLOCK_REALTIME",
-                  "clock_gettime((clockid_t)CLOCK_REALTIME", false
+                  "clock_gettime((clockid_t)CLOCK_REALTIME"
+          s.gsub! "clock_getres(CLOCK_REALTIME",
+                  "clock_getres((clockid_t)CLOCK_REALTIME", false
         end
-      end
 
-      # Fix "candidate function not viable: no known conversion from
-      # 'folly::detail::Clock' to 'clockid_t' for 1st argument"
-      # See upstream PR mentioned above
-      if MacOS.version == "10.11" && MacOS::Xcode.installed? && MacOS::Xcode.version >= "8.0"
+        # Fix "candidate function not viable: no known conversion from
+        # 'folly::detail::Clock' to 'clockid_t' for 1st argument"
+        # See upstream PR mentioned above
         inreplace "portability/Time.h", "typedef uint8_t clockid_t;", ""
```

Comment:
What is `clockid_t` being defined as on 10.11? `clock_gettime` shouldn't exist on 10.11, should it?